global.json

```
@@ -3,13 +3,13 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "9.0.100-rc.2.24474.11",
+    "dotnet": "9.0.100",
```

Should we move ahead faster than Arcade?
```suggestion
    "dotnet": "9.0.101",
```

I don't have super strong feelings either way.

---

I'd prefer conservative approach now, butI might be missing what changes or analyzers would this bring (or would it be only to catch gotchas early?)

---

Yeah just dogfooding, no urgent need AFAIK.